youtube_dl/extractor/threeplus.py

```diff
@@ -0,0 +1,102 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from .threeqsdn import ThreeQSDNIE
+from ..utils import get_element_by_class
+
+
+class ThreePlusBaseIE(InfoExtractor):
+    _HOST_URL = 'http://playout.3qsdn.com/'
+
+    def _get_title_and_description(self, video_id):
+        webpage = self._download_webpage(
+            self._HOST_URL + video_id, video_id)
+        title = self._og_search_title(webpage)
+        description = self._og_search_description(webpage)
+        return title, description
+
+    def _get_real_video_id(self, video_id):
+        video_info = self._download_webpage(self._HOST_URL + video_id + '?js=true', video_id)
```

Comment:
I assumed that but was afraid, that I would break other stuff depending on 3qsdn. [Here](http://playout.3qsdn.com/b964ee16-ed42-11e6-a78b-0cc47a188158?js=true) is an example script. My code searches for the first appearing field key `sdnPlayoutId` and extracts its value. Should I do exactly the same, but in the 3qsdn extractor?